databricks-sdk-java/src/test/java/com/databricks/sdk/integration/FilesIT.java

```diff
@@ -46,10 +45,6 @@ private void writeFileAndReadFileInner(
     // Write the file to DBFS.
     workspace.files().upload(fileName, inputStream);
 
-    // Read file status
-    FileInfo fileStatus = workspace.files().getStatus(fileName);
-    Assertions.assertEquals(fileStatus.getPath(), fileName);
-
```

Comment:
Get status doesn't work. Same reason as for the python sdk [here](https://github.com/databricks/databricks-sdk-py/pull/444/files#diff-a4a7d7aa2604f48fbe447a1274bcf9aeb9d1805e4087ac4d036d8fd6d44ff230L225-L227), removing these two lines.